lighthouse-plugin-publisher-ads/audits/ad-blocking-tasks.js

```diff
@@ -26,8 +26,7 @@ const UIStrings = {
   'performance on less powerful devices. [Learn more](' +
   'https://developers.google.com/publisher-ads-audits/reference/audits/ad-blocking-tasks' +
   ').',
-  failureDisplayValue: '{numTasks, plural, =1 {1 long task} other ' +
-  '{# long tasks}}',
+  failureDisplayValue: '{timeInMs, number, seconds} s blocked',
```

Comment:
Can you run messages/collect-strings.js to update the strings? Thinking about it now, this should be part of our precommit.